README.md

```diff
@@ -50,7 +50,7 @@ With the following configuration options set as environment variables:
 * `LEDGER_HOSTNAME` (default: *[your hostname]*) Publicly visible hostname. This is important for things like generating globally unique IDs. Make sure this is a hostname that all your clients will be able to see. The default should be fine for local testing.
 * `LEDGER_PUBLIC_PORT` (default: `$PORT`) Publicly visible port. You can set this if your public port differs from the listening port, e.g. because the ledger is running behind a proxy.
 * `LEDGER_PUBLIC_HTTPS` (default: `''`) Whether or not the publicly visible instance of Five Bells Ledger is using HTTPS.
-* `LEDGER_ADMIN_USER` (default: `'admin'`) The admin account's username (an admin user can create/modify accounts).
+* `LEDGER_ADMIN_NAME` (default: `'admin'`) The admin account's username (an admin user can create/modify accounts).
```

Comment:
Is this right? It looks like the code uses `LEDGER_ADMIN_USER`: https://github.com/interledger/five-bells-ledger/blob/master/src/lib/config.js#L35